appveyor.yml

```diff
@@ -4,21 +4,21 @@ environment:
   - MSYS_BITS: 64
     TARGET: x86_64-pc-windows-msvc
     CHECK: check
-    CONFIGURE_ARGS: --enable-llvm-assertions --enable-debug-assertions
+    CONFIGURE_ARGS: --enable-vendor
```

Comment:
Oh this is where the windows/travis builds differ ever so slightly. The Travis builds go through `src/ci/run.sh` which unconditionally adds `--enable-vendor`. The AppVeyor builds, however, do configure + make manually (avoiding `src/ci/run.sh`) which means we have to duplicate a bit of logic, and is why `--enable-vendor` is explicitly spelled out here.

I can look into unifying these two paths, though.